agent/rpc/middleware/interceptors.go

```diff
@@ -42,9 +44,8 @@ func (r *RequestRecorder) Record(requestName string, rpcType string, start time.
 		{Name: "rpc_type", Value: rpcType},
 	}
 
-	// TODO(rpc-metrics-improv): consider using Telemetry API call here
-	// It'd be neat if we could actually pass the elapsed observed above
-	metrics.MeasureSinceWithLabels(metricRPCRequest, start, labels)
+	// TODO(FFMMM): it'd be neat if we could actually pass the elapsed observed above
```

Comment:
This would require a change in `go-metrics` to support an "elapsed" time concept when recording.

Since this is not particularly tied to the effort, I reclassified the todo

Reply:
Doesn't `metrics.MeasureSinceWithLabels` measure elapsed? I thought that's what `start` was for